RevertConditions/ReturnSize/README.md

```diff
@@ -0,0 +1,28 @@
+# Revert On Return Size Example
+
+## Overview
+This repository contains a smart contract named `ReturnSize` that includes a function called `callerFunction`. This function takes a library address and a uint as arguments, relying on the implementation of `calledFunction` within the provided library address to execute certain logic based on the input uint. The purpose of this example is to demonstrate the utilization of the CVL prover argument flag `-superOptimisticReturnsize`. This flag ensures that the correct number of return values is obtained from a NONDET (non-deterministic) unresolved call, thereby preventing a revert scenario in such instances.
```

Comment:
library is a bit confusing here - it is an external contract. right?

The purpose of this example is to show a revert cases due to external call return size.
Solidity compiler adds an internal check to validate the return size 

(we can document the flag and link to this example for the cliflags folder)